presto-main/src/main/java/com/facebook/presto/cost/StatsNormalizer.java

```diff
@@ -87,8 +86,8 @@ private PlanNodeStatsEstimate normalize(PlanNodeStatsEstimate stats, Optional<Co
      */
     private SymbolStatsEstimate normalizeSymbolStats(Symbol symbol, SymbolStatsEstimate symbolStats, PlanNodeStatsEstimate stats, TypeProvider types)
     {
-        if (UNKNOWN_STATS.equals(symbolStats)) {
-            return UNKNOWN_STATS;
+        if (symbolStats.isUnknown()) {
+            return SymbolStatsEstimate.unknown();
```

Comment:
After this change, there is no `UNKNOWN_STATS` constant, so you could as well return `symbolStats`

Reply:
Returning `SymbolStatsEstimate.unknown()` is more explicit. Think of it as of

```
if(variable == null){
  return null;
}
```